drivers/windows/pandaJ2534DLL/PandaJ2534Device.cpp

```diff
@@ -67,11 +74,28 @@ DWORD PandaJ2534Device::addChannel(std::shared_ptr<J2534Connection>& conn, unsig
 }
 
 DWORD PandaJ2534Device::can_recv_thread() {
```

Comment:
Separated reading USB data from message processing to prevent data reading and message processing from getting behind